### PR TITLE
Update Wachowskis entry

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR updates the entry on the Wachowskis as suggested in #2. It uses the first image referenced in the comment. This image is about 60kB larger than the previous one. If this is an issue, let me know, and I'd be happy to go and find a different one or compress this one. If the image is an issue, I'm also happy to exclude that from the PR and just update the name for now. 

Any other input or feedback is also much appreciated! 🙏🏾

### CODE REVIEW

**Comment by sigdestad** [comment]
Hmm.. Rather than manipulating the export files inside the code, you should rather import the content in a local instance, modify it than export the result and place it in the app. That way everything will be peaches in terms of consistency.

**Comment by thomasheartman** [comment]
Sure; that sounds good! I’m not entirely sure how to do what you’re describing, though. Is there a guide for this? Or will it become obvious once I start? I’m not at my computer right now, so it’s a bit abstract at the moment. Thanks for the quick feedback ☺️

**Comment by sigdestad** [comment]
Just Install hmdb, modify the content and export with data toolbox

**Comment by thomasheartman** [comment]
Got it! Just to make  sure I got it right:

In data toolbox, you can use the 'data tree' option to find the project you want. Navigate to the 'draft' branch (master was empty) and export the content. You can then copy this into the source repo where you got the content from originally. 

Seems to work alright, except that for a few issues that I'm hoping you can help me with.

At first it seemed that I had changed a bunch of permissions. In more or less every `node.xml` file, I had a diff that said I added new users with privileges and removed `cms.cm.app`:

<details>

<summary>Permissions changes</summary>

```diff
@@ -5,12 +5,24 @@
     <timestamp>2020-09-23T12:34:46.777Z</timestamp>
     <inheritPermissions>true</inheritPermissions>
     <permissions>
-        <principal key="role:cms.cm.app">
+        <principal key="role:cms.project.hmdb.viewer">
             <allow type="array">
                 <value>READ</value>
             </allow>
             <deny type="array"/>
         </principal>
+        <principal key="role:cms.project.hmdb.owner">
+            <allow type="array">
+                <value>READ</value>
+                <value>CREATE</value>
+                <value>MODIFY</value>
+                <value>DELETE</value>
+                <value>PUBLISH</value>
+                <value>READ_PERMISSIONS</value>
+                <value>WRITE_PERMISSIONS</value>
+            </allow>
+            <deny type="array"/>
+        </principal>
         <principal key="role:system.admin">
             <allow type="array">
                 <value>READ</value>
@@ -23,6 +35,18 @@
             </allow>
             <deny type="array"/>
         </principal>
+        <principal key="role:cms.project.hmdb.editor">
+            <allow type="array">
+                <value>READ</value>
+                <value>CREATE</value>
+                <value>MODIFY</value>
+                <value>DELETE</value>
+                <value>PUBLISH</value>
+                <value>READ_PERMISSIONS</value>
+                <value>WRITE_PERMISSIONS</value>
+            </allow>
+            <deny type="array"/>
+        </principal>
         <principal key="role:cms.admin">
             <allow type="array">
                 <value>READ</value>
@@ -35,6 +59,27 @@
             </allow>
             <deny type="array"/>
         </principal>
+        <principal key="role:cms.project.hmdb.contributor">
+            <allow type="array">
+                <value>READ</value>
+            </allow>
+            <deny type="array"/>
+        </principal>
+        <principal key="role:cms.project.hmdb.author">
+            <allow type="array">
+                <value>READ</value>
+                <value>CREATE</value>
+                <value>MODIFY</value>
+                <value>DELETE</value>
+            </allow>
+            <deny type="array"/>
+        </principal>
+        <principal key="role:system.everyone">
+            <allow type="array">
+                <value>READ</value>
+            </allow>
+            <deny type="array"/>
+        </principal>
     </permissions>
     <data>
         <boolean name="valid">true</boolean>
```

</details>

This seems to suggest that I've added a bunch of new roles (or even users). I managed to get around this by changing the permissions structure according to the of the root node (not inheriting permissions, removing hmbd users). 

Now, every node.xml (or at least 70 of them; I'm assuming that's the lot), has updated timestamps because of permissions changes (presumably):
 
```diff

     <id>c2998dba-c56e-491e-af36-0ffc4a74c67a</id>
     <childOrder>modifiedtime DESC</childOrder>
     <nodeType>content</nodeType>
-    <timestamp>2020-09-23T12:34:46.762Z</timestamp>
+    <timestamp>2021-03-17T09:09:49.583Z</timestamp>
     <inheritPermissions>true</inheritPermissions>
     <permissions>
         <principal key="role:cms.cm.app">
```

Should I include this or can I discard those changes? Is there a better way to do this? I spun up the app in a clean sandbox.

**Comment by sigdestad** [comment]
Using inherit = true and cleaning away other permissions sound correct to me. Also new timestap is no problem I guess.

Just create a new branch and I can give it a spin before we merge to master.

**Comment by thomasheartman** [comment]
Thanks for the input. Here's what I've done now.

I've broken up the changes into three commits. 

1. The first one (7442f131) contains what I consider the essential changes. From my little testing, it seems to work as expected. If this does indeed work, I suggest we discard any later commits and use only this.

2. The second one (a9c4c5a) contains what I consider to be accidental changes to the Wachowskis' entry. This is stuff such as workflow and SoMe property sets, `pathIndexConfig`, etc. I don't know whether this is necessary, but it seems superfluous and is probably a result of me touching some fields I shouldn't have. If these changes are necessary and sufficient, I suggest we use this commit or, if desired, break it into smaller pieces (if there are some of these changes that we want, and others not).

3. The third commit is the one that updates the timestamp on all the nodes. It touches 69 files and I don't think it is important. Again, I think it's just a function of me touching some controls in an unfortunate manner.

So to recap: If it's sufficient, let's merge only the first commit. If necessary, we'll add the second one. If that's not enough, let's consider the third one. Does that sound alright?

**Comment by thomasheartman** [comment]
@tsi?

**Comment by thomasheartman** [comment]
@tsi Any updates on this? Are you still looking? 👀

**Comment by tsi** [comment]
@thomasheartman you must be confusing me with someone else :)

**Comment by thomasheartman** [comment]
@tsi, oh, no! I'm so sorry 🙈  We use 3-letter identifiers internally, and `tsi` is @sigdestad's 😅  

So indeed, @sigdestad: where are we at with this?

**Comment by sigdestad** [comment]
Maybe you could rather demo this for me, and we could conclude in meeting?
